feature/bgp/addpath/ate_tests/route_propagation_test/route_propagation_test.go

```diff
@@ -121,6 +121,12 @@ func (d *dutData) Configure(t *testing.T, dut *ondatra.DUTDevice) {
 		ocName := dut.Port(t, a.Name).Name()
 		dut.Config().Interface(ocName).Replace(t, a.NewInterface(ocName))
 	}
+
+	t.Log("Configure Network Instance")
+	dutConfNIPath := dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance)
+	dutConfNIPath.Type().Replace(t, telemetry.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)
+	dutConfNIPath.RouterId().Replace(t, dutPort2.IPv4)
+
```

Comment:
use BGP router id as in https://github.com/openconfig/featureprofiles/pull/567

Reply:
Done. Thanks.